### PR TITLE
feat: Use new small cross and small plus icons

### PR DESCRIPTION
**Pull request by Crash--**
wait for https://github.com/cozy/cozy-ui/pull/818

### CODE REVIEW

**Comment by Crash--** [comment]
[Contacts - formulaire création/édition - pictos](https://trello.com/c/IkFyodjT/1566-contacts-formulaire-cr%C3%A9ation-%C3%A9dition-pictos)

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
   "homepage": "https://github.com/cozy/cozy-contacts#readme",
   "devDependencies": {
     "babel-preset-cozy-app": "1.3.0",
-    "cozy-scripts": "1.12.1",
+    "cozy-scripts": "1.11.0",
```
Comment (y-lohse) [inline package.json]: Why are we downgrading cozy-scripts in this PR?

File: src/components/ContactCard/ContactForm/ContactFormField.jsx [hunk 2]
```diff
@@ -82,7 +82,7 @@ class ContactFormField extends React.Component {
             onClick={() => this.addField(fields)}
             className="contact-form__meta-button contact-form__meta-button--add"
           >
-            <Icon icon={'plus'} color="white" width="10" height="10" />
+            <Icon icon={'plus-small'} color="white" width="10" height="10" />
```
Comment (y-lohse) [inline src/components/ContactCard/ContactForm/ContactFormField.jsx]: For the plus icon above, we changed the size from 10 to 12, but not here. Is it normal?

File: package.json (hunk 1 above)
Comment (Crash--) [reply package.json]: See the chan. renovate has updated CS during the night and we has an issue with this release. 

We expect a new release today / tomorrow.

File: src/components/ContactCard/ContactForm/ContactFormField.jsx (hunk 2 above)
Comment (Crash--) [reply src/components/ContactCard/ContactForm/ContactFormField.jsx]: Yep, it looks great that way ;).

File: package.json (hunk 1 above)
Comment (y-lohse) [reply package.json]: Mmh alright, but renovate is going to autoupdate it again, and if the newest version isn't out by then we'll have the same problem.

File: src/components/ContactCard/ContactForm/ContactFormField.jsx (hunk 2 above)
Comment (y-lohse) [reply src/components/ContactCard/ContactForm/ContactFormField.jsx]: Best reason 😎

**Pull request merged**